examples/ReactJsSdkApp/public/index.html

```diff
@@ -26,13 +26,14 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
-    <script async defer src="https://sdk.apidev.pticlient.com/0.0.10/index.js"></script>
+    <script async defer src="https://sdk.apidev.pticlient.com/0.0.11/index.js"></script>
     <script>
+      ptiConfig = {
+        clientId: "3450582c-1955-11eb-adc1-0242ac120002",
```

Comment:
Do we really want to have the test cient id here?

Reply:
Added comment to explain what this value means